pkg/vdri/key/creator.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package key
+
+import (
+	"encoding/binary"
+	"fmt"
+	"time"
+
+	"github.com/btcsuite/btcutil/base58"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	vdriapi "github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdri"
+	"github.com/hyperledger/aries-framework-go/pkg/internal/cryptoutil"
+)
+
+const (
+	schemaV011                 = "https://w3id.org/did/v0.11"
```

Comment:
why the old schema?

https://w3c-ccg.github.io/did-method-key/ has https://w3id.org/did/v1

Reply:
I used https://uniresolver.io/ as a reference and they are using that schema. Will update to the newest one as per spec.

Reply:
Yes, thanks. Let's ensure we are using the up-to-date spec.

Reply:
You should also review the spec for any other differences.

Reply:
done